content/reference-strategy/index.md

```diff
@@ -0,0 +1,173 @@
+* It's not possible to infer a type constraint for any new value.
+
+
+## Explanation
+
+Although Registers could be built on any of the exposed solutions, a CURIE has
+the edge in flexibility and builds on top of existing open standards for
+linking resources.
+
+### Expanding a CURIE
+
+The current solution for expanding CURIEs relies on a few assumptions. In
+order to remove this RFC proposes providing a **Context**, in a similar way
+[JSON-LD](https://json-ld.org/) does it.
+
+The definition of the context will happen at the Register Environment level,
+this way it can be guaranteed that a single context will work for any Register
+part of the Environment.
+
+A Context is a dictionary of prefixes and URIs, in JSON it would look like:
+
+```json
+{
+  "country": "https://country.register.gov.uk/records/",
+  "lae": "https://local-authority-eng.register.gov.uk/records/",
+  "example": "https://example.org/"
+}
+```
+
+With this context, the following expansions are expected to be true:
+
+* `country:GB` -> `https://country.register.gov.uk/records/GB`
+* `country:ZZ` -> `https://country.register.gov.uk/records/ZZ`
+* `lae:BIR` -> `https://local-authority-eng.register.gov.uk/records/BIR`
+* `example:32` -> `https://example.org/32`
+
+Note that currently the context is managed partially by the Register register.
+Further work is expected to evolve it to match this definition of context.
+
+## Naming conventions
+
+Foreign keys have been using the same field definition as the primary key in
+the targeted register. This means that a sigle entry in the field register
+defines a field used in one register as the primary key and in another
+register as a foreign key. This convention derives from another one where the
+primary key field is always the same as the register identifier. E.g. the
+register with id `country`, has a field `country` that acts as the primary key
+for that register. Another register with a field `country` will use the field
+as a foreign key.
+
+With CURIEs this assumption weakens. A future RFC will define the new
+convention for naming fields of type CURIE although this will be _informative_
+not _normative_. The current thinking is to move away from using names and
+start using relationships. For example, given a register with a link to the
+organisation that manages the resource, it would use something like
+`managed-by` instead of `manager` or `organisation`.
+
+A future RFC will formalise the way to explicitly identify the primary key for
+a register so the convention of using the register identifier will be
+addressed there.
+
+## Conclusion
+
+This RFC accepts CURIE as the only mechanism for linking between register from
+now on given they offer the flexibility required by registers.
+
+Existing registers that use foreign keys will be maintained to avoid
+disruption in the registers ecosystem. An independent task will define and
+execute the migration from foreing keys to CURIEs.
```

Comment:
there's a typo here: "foreing"

Reply:
actually you could just drop this whole sentence as it doesn't say when this will happen or how.